python-ovirt-engine-sdk4.spec.in

```diff
@@ -28,16 +28,25 @@ This package contains the Python 3 SDK for version 4 of the oVirt Engine
 API.
 
 %if 0%{?rhel} < 9
-%package -n python38-ovirt-engine-sdk4
+
+# For el8, build for the python version that ansible-core uses,
+# so that our ansible code can use the sdk.
+# This requires ansible-core to be installed beforehand.
+%global ansible_python_bin %(rpm -q --requires ansible-core | grep /usr/bin/python)
+%global ansible_python_rpm %(rpm -q --qf '%{NAME}' -f "%{ansible_python_bin}"
+%global ansible_python_major_dot_minor %(%{ansible_python_bin} -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')
+%global ansible_python_major_minor %(%{ansible_python_bin} -c 'import sys; print(f"{sys.version_info.major}{sys.version_info.minor}")')
```

Comment:
Wouldn't it be easier to rely on `python-abi` requirements?
```
rpm -q --requires ansible-core | grep 'python(abi)' | sed 's/python(abi) = //'
```

Reply:
Not sure it's significant. Done, anyway.